### PR TITLE
Add metadata section to return json in Retrieve governance connector, GET endpoint. 

### PR DESCRIPTION
**Pull request by ravindudsilva**
## Purpose
Add metadata section to return json in ``Retrieve governance connector`` , GET endpoint. 

### Changes in this pull request
Updated openApi 3.0 definition in ``src/main/resources/identity-governance.yaml`` and regenerated the API of  `` org.wso2.carbon.identity.api.server.identity.governance.v1 `` and passed the metadata properties.

New Response
```
  "properties": [
        { "name": "Recovery.AdminPasswordReset.RecoveryLink",
            "value": "false",
            "displayName": "Enable password reset via recovery e-mail",
            "description": "User gets notified with a link to reset password",
            "meta": {
                "type": "boolean"
            }
       }
 ]
```

Related issue [10084](https://github.com/wso2/product-is/issues/10084)

### CODE REVIEW

File: components/org.wso2.carbon.identity.api.server.identity.governance/org.wso2.carbon.identity.api.server.identity.governance.v1/src/gen/java/org/wso2/carbon/identity/api/server/identity/governance/v1/model/MetaRes.java [hunk 1]
```diff
@@ -0,0 +1,100 @@
+/*
+* Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (somindatommy) [inline components/org.wso2.carbon.identity.api.server.identity.governance/org.wso2.carbon.identity.api.server.identity.governance.v1/src/gen/java/org/wso2/carbon/identity/api/server/identity/governance/v1/model/MetaRes.java:2]: Let's update the year correctly. You can refer to [1].
[1] - https://somindagamage.medium.com/adding-and-using-wso2-copyright-license-in-intellij-idea-8d172ce8c420

File: components/org.wso2.carbon.identity.api.server.identity.governance/org.wso2.carbon.identity.api.server.identity.governance.v1/src/gen/java/org/wso2/carbon/identity/api/server/identity/governance/v1/model/PropertyRes.java [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /*
-* Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (somindatommy) [inline components/org.wso2.carbon.identity.api.server.identity.governance/org.wso2.carbon.identity.api.server.identity.governance.v1/src/gen/java/org/wso2/carbon/identity/api/server/identity/governance/v1/model/PropertyRes.java:2]: Lets not change the license of existing files

File: components/org.wso2.carbon.identity.api.server.identity.governance/org.wso2.carbon.identity.api.server.identity.governance.v1/src/main/resources/identity-governance.yaml [hunk 3]
```diff
@@ -643,7 +646,8 @@ components:
         self:
           type: string
           description: Path to retrieve the full connector information.
-          example: /t/carbon.super/api/server/v1/identity-governance/QWNjb3VudCBNYW5hZ2VtZW50IFBvbGljaWVz/connectors/c3VzcGVuc2lvbi5ub3RpZmljYXRpb24
```
Comment (somindatommy) [inline components/org.wso2.carbon.identity.api.server.identity.governance/org.wso2.carbon.identity.api.server.identity.governance.v1/src/main/resources/identity-governance.yaml:646]: Any reason to change this?

File: components/org.wso2.carbon.identity.api.server.identity.governance/org.wso2.carbon.identity.api.server.identity.governance.v1/src/main/resources/identity-governance.yaml [hunk 4]
```diff
@@ -710,6 +714,18 @@ components:
           type: string
           description: Property description.
           example: Enable account suspend notifications.
+        meta:
+          type: object
+          description: Meta data properties
```
Comment (somindatommy) [inline components/org.wso2.carbon.identity.api.server.identity.governance/org.wso2.carbon.identity.api.server.identity.governance.v1/src/main/resources/identity-governance.yaml:719]: Lets update as `Meta Data related to each property`

**Comment by somindatommy** [comment]
@Ammoniya make sure adding the meta-object will not affect any other API response or request. Lets try the flows and verify that

File: components/org.wso2.carbon.identity.api.server.identity.governance/org.wso2.carbon.identity.api.server.identity.governance.v1/src/main/resources/identity-governance.yaml (hunk 4 above)
Comment (Ammoniya) [reply components/org.wso2.carbon.identity.api.server.identity.governance/org.wso2.carbon.identity.api.server.identity.governance.v1/src/main/resources/identity-governance.yaml:718]: Fixed in commit 37c3b01b50911acef42cdeb84eb9d2405b1c1c71

File: components/org.wso2.carbon.identity.api.server.identity.governance/org.wso2.carbon.identity.api.server.identity.governance.v1/src/main/resources/identity-governance.yaml (hunk 3 above)
Comment (Ammoniya) [reply components/org.wso2.carbon.identity.api.server.identity.governance/org.wso2.carbon.identity.api.server.identity.governance.v1/src/main/resources/identity-governance.yaml]: Fixed in commit 37c3b01b50911acef42cdeb84eb9d2405b1c1c71

File: components/org.wso2.carbon.identity.api.server.identity.governance/org.wso2.carbon.identity.api.server.identity.governance.v1/src/gen/java/org/wso2/carbon/identity/api/server/identity/governance/v1/model/PropertyRes.java (hunk 2 above)
Comment (Ammoniya) [reply components/org.wso2.carbon.identity.api.server.identity.governance/org.wso2.carbon.identity.api.server.identity.governance.v1/src/gen/java/org/wso2/carbon/identity/api/server/identity/governance/v1/model/PropertyRes.java]: Fixed in commit 37c3b01b50911acef42cdeb84eb9d2405b1c1c71

File: components/org.wso2.carbon.identity.api.server.identity.governance/org.wso2.carbon.identity.api.server.identity.governance.v1/src/gen/java/org/wso2/carbon/identity/api/server/identity/governance/v1/model/MetaRes.java (hunk 1 above)
Comment (Ammoniya) [reply components/org.wso2.carbon.identity.api.server.identity.governance/org.wso2.carbon.identity.api.server.identity.governance.v1/src/gen/java/org/wso2/carbon/identity/api/server/identity/governance/v1/model/MetaRes.java:2]: Fixed in commit 37c3b01b50911acef42cdeb84eb9d2405b1c1c71

**Review by somindatommy — APPROVED** [review]

**Comment by Ammoniya** [comment]
New response after adding ``groupID``  to meta data section return JSON.

```       {
            "name": "SelfRegistration.CallbackRegex",
            "value": "[https://localhost:9443].*[/authenticationendpoint/login.do]",
            "displayName": "User self registration callback URL regex",
            "description": "This prefix will be used to validate the callback URL.",
            "meta": {
                "type": "string",
                "regex": ".*",
                "groupID": 0
            }
        },

File: components/org.wso2.carbon.identity.api.server.identity.governance/org.wso2.carbon.identity.api.server.identity.governance.v1/src/gen/java/org/wso2/carbon/identity/api/server/identity/governance/v1/model/MetaRes.java (hunk 1 above)
Comment (IsuraD) [reply components/org.wso2.carbon.identity.api.server.identity.governance/org.wso2.carbon.identity.api.server.identity.governance.v1/src/gen/java/org/wso2/carbon/identity/api/server/identity/governance/v1/model/MetaRes.java:2]: make it 2021

**Review by IsuraD — APPROVED** [review]

**Comment by somindatommy** [comment]
Closing this since this is resolved by https://github.com/wso2/identity-api-server/pull/288

**Pull request closed without merging**